velox/type/TimestampConversion.cpp

```diff
@@ -593,6 +613,28 @@ Status daysSinceEpochFromWeekDate(
   return Status::OK();
 }
 
+Expected<int64_t> daysSinceEpochFromWeekOfMonthDate(
+    int32_t year,
+    int32_t month,
+    int32_t weekOfMonth,
+    int32_t dayOfWeek) {
+  if (!isValidWeekOfMonthDate(year, month, weekOfMonth, dayOfWeek)) {
+    return folly::makeUnexpected(Status::UserError(
```

Comment:
let's use threadSkipErrorDetails to generate error message only if needed, see https://velox-lib.io/blog/optimize-try-more#do-not-produce-or-store-error-messages-under-try

Reply:
Updated. Thanks.